File: Stephannie/Exercise-10.py
normal_year = 365
leap_year = 366

year = int(input("Enter the year you want to check: "))

condition_1 = year/4
condition_2 = year/100
condition_3 = year / 400

if condition_1 % 2 == 0 and condition_2 % 2 != 0:
    print("The year is a leap year")

elif condition_1 % 2 == 0 and condition_3 % 2 ==0:
    print("The year is a leap year")

elif condition_2 % 2 != 0: 
    print("The year is not a leap year.")

elif condition_3 % 2 != 0:
    print("The year is not a leap year.")

else:
    print("The year you entered is not tracked")

    

File: Pokua/Exercise10.java
import java.util.Scanner;

public class Exercise10{

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Enter a year: ");
        int year = input.nextInt();

        if (year % 4 == 0 && year % 100 != 0 || year % 400 == 0) {
            System.out.println("leap year");
        }
        else {
            System.out.println("not a leap year");
        }

        input.close();
    }

}
